src/Attributable.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace SilverStripe\Assets;
+
+use SilverStripe\Core\Convert;
+
+trait Attributable
```

Comment:
This trait is made up of code copied from FormField. I've got here for now, but if we go ahead with this, I would put it in Framework

The idea is that a bunch of classes use this attribute handling logic. By putting in one trait, we'll be bale to have a standardised pattern we can reuse across a bunch of classes.

This is a bit of retroactive class composition. Not overly fuss about this, we could just put all that logic directly on `ImageManipulation'

Reply:
If we stick with the trait, I'd rename it to something like HTMLAttributes or similar; Attributable sounds more like an interface suggesting the class using it can be attributed to something/someone, which probably isn't the case here.

Reply:
Yeah ... not a big fan of Attributable name. I pick it up quickly just to get me unblock. Looks like we'll do a separate PR for this anyway.

Reply:
Got a follow up PR and issue https://github.com/silverstripe/silverstripe-framework/pull/10033